src/Views/Form/Templates/Classic/resources/js/form.js

```diff
@@ -226,14 +225,18 @@ function addTooltipToLevel(node) {
     const parent = node.parentNode;
     if (!node.getAttribute('has-tooltip')) {
         const tooltip = nodeFromString(
-            h('span', {className: 'give-tooltip hint--top hint--bounce', 'aria-label': node.innerHTML})
+            h('span', {
+                className: 'give-tooltip hint--top hint--bounce',
+                'aria-label': parent.getAttribute('aria-label'),
+            })
         );
         if (node.innerHTML.length < 50) {
             tooltip.classList.add('narrow');
         }
         parent.replaceChild(tooltip, node);
         tooltip.appendChild(node);
         node.setAttribute('has-tooltip', 'true');
+        console.log(parent.ariaLabel);
```

Comment:
@JoshuaHungDinh please remove this console log

Reply:
Thanks you!
 Resolved: https://github.com/impress-org/givewp/pull/6653/commits/c5456e41be871abd9a522b1766fe6e05c5fc97c1